### PR TITLE
cloud: add 20240220 release notes

### PR DESCRIPTION
**Pull request by hfxsd**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v8.0 (TiDB 8.0 versions)
- [ ] v7.6 (TiDB 7.6 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/tidb-cloud-release-notes.md [hunk 1]
```diff
@@ -8,6 +8,15 @@ aliases: ['/tidbcloud/supported-tidb-versions','/tidbcloud/release-notes']
 
 This page lists the release notes of [TiDB Cloud](https://www.pingcap.com/tidb-cloud/) in 2024.
 
+## February 20, 2024
+
+**General changes**
+
+- Support creating more TiDB Cloud nodes on Google Cloud.
+
+    - If you set a regional CIDR size of `/19` for Google Cloud, you can create up to 128 TiDB Cloud nodes on Google Cloud in any region of a project.
```
Comment (hfxsd) [inline tidb-cloud/tidb-cloud-release-notes.md:17]:
```suggestion
    - If you set a regional CIDR size of `/19` for Google Cloud, you can create up to 128 TiDB Cloud nodes on Google Cloud within any region of a project.
```

File: tidb-cloud/tidb-cloud-release-notes.md [hunk 2]
```diff
@@ -8,6 +8,15 @@ aliases: ['/tidbcloud/supported-tidb-versions','/tidbcloud/release-notes']
...
 This page lists the release notes of [TiDB Cloud](https://www.pingcap.com/tidb-cloud/) in 2024.
 
+## February 20, 2024
+
+**General changes**
+
+- Support creating more TiDB Cloud nodes on Google Cloud.
+
+    - If you set a regional CIDR size of `/19` for Google Cloud, you can create up to 128 TiDB Cloud nodes on Google Cloud within any region of a project.
+    - If you want to create more than 128 nodes in any region of a project, you can contact [TiDB Cloud Support](/tidb-cloud/tidb-cloud-support.md) to configure an IP range size between `/16` and `/18`.
```
Comment (hfxsd) [inline tidb-cloud/tidb-cloud-release-notes.md:18]:
```suggestion
    - By configuring a regional CIDR size of `/19` for Google Cloud, you can now create up to 128 TiDB Cloud nodes within any region of a project.
    - If you want to create more than 128 nodes in any region of a project, you can contact [TiDB Cloud Support](/tidb-cloud/tidb-cloud-support.md) for assistance in customizing an IP range size ranging from `/16` to `/18`.
```

**Review by qqqdan — APPROVED** [review]

File: tidb-cloud/tidb-cloud-release-notes.md [hunk 3]
```diff
@@ -8,6 +8,15 @@ aliases: ['/tidbcloud/supported-tidb-versions','/tidbcloud/release-notes']
 
 This page lists the release notes of [TiDB Cloud](https://www.pingcap.com/tidb-cloud/) in 2024.
 
+## February 20, 2024
+
+**General changes**
+
+- Support creating more TiDB Cloud nodes on Google Cloud.
+
+    - By configuring a regional CIDR size of `/19` for Google Cloud, you can now create up to 128 TiDB Cloud nodes within any region of a project.
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-release-notes.md:17]:
```suggestion
    - By [configuring a regional CIDR size](/tidb-cloud/set-up-vpc-peering-connections.md#prerequisite-set-a-cidr-for-a-region) of `/19` for Google Cloud, you can now create up to 128 TiDB Cloud nodes within any region of a project.
```

**Review by qiancai — APPROVED** [review]
Rest LGTM

**Pull request merged**